### DISCUSSION TITLE
FR: Ride music cover art & trivia.

### DISCUSSION THREAD

**Discussion by karst** [OpenRCT2/OpenRCT2#17385] [Feature requests] [answered]
I would love to see the ability for a preview image in an audio `parkobj`  file. Having all original RCT & RCT2 tracks have the album art of their respective games. Have all OpenRCT2 official music have a cover art that fits the theme with the OpenRCT2 logo on it. 

If this would be added this would only be visible on Object Selection. To make its presence better, on the ride music selection we would also be able to see the preview image, aswell as trivia.

Quick mockup:
![image](https://user-images.githubusercontent.com/1516893/173710442-29d39085-e756-4cc0-94bd-38c79d7ad72a.png)

Trivia would be the track names and artist. And author on the bottom right.

The album image would be in the place where I currently have the merrie go round image.

Additionally it would be neat if stalls could play ride music and if there'd be a scenery item that'd have a UI to sync music with a music group of which a stall or ride is main of to extend the radius of its effect.

**Comment by TheAtomTwister** [comment]
That's a pretty interesting idea.  I like the idea of tinkering with the music in the game.

**Comment by karst** [comment, marked as answer]
Implemented as of #20832
